doordeck-sdk/src/mingwMain/resources/csharp/Doordeck.Headless.Sdk.csproj

```diff
@@ -0,0 +1,10 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <TargetFramework>net9.0</TargetFramework>
```

Comment:
You've got dotnet 8.x in the setup-dotnet gradle action, should it be 9 or should this be 8?

Reply:
9 is the right version, thanks!